Makefile

```diff
@@ -179,6 +179,10 @@ unit:
 run-ci-e2e-test:
 	hack/run-ci-e2e-test.sh -t basic
 
+.PHONY: run-ci-e2e-byoh-test
+run-ci-e2e-byoh-test:
+	hack/run-ci-e2e-test.sh -t basic -m 0
+
```

Comment:
```suggestion
.PHONY: run-ci-e2e-byoh-test $(ARGS)
run-ci-e2e-byoh-test:
        if [ -z "${ARGS}" ]; then hack/run-ci-e2e-test.sh -t basic; else hack/run-ci-e2e-test.sh ${ARGS}; fi
```

Reply:
Thoughts on this? Prevents having to maintain 2 make targets, and any edits to release repo

Reply:
From previous [comments](https://github.com/openshift/windows-machine-config-operator/pull/858#discussion_r781357536):

> IOW, don't introduce $(ARGS).

>  we want to reduce the interaction we have with the release repo



@saifshaikh48, the point here is to minimize the maintenance on the release repo, so that WMCO' e2e tests are configured in WMCO.

Reply:
>we will eventually get rid of run-ci-e2e-byoh-test

I'd prefer the one target with a conditional like Mohammad suggested, but I'm ok with the two targets for now to minimize release repo maintenance

Reply:
It does not.

> one target with a conditional

Moves the WMCO' e2e tests configuration to the release repo, and encourages more maintenance on the release repo side.

Reply:
I agree with @jrvaldes on keeping the release repo simple